configuration/managing-preview-features.adoc

```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="managing-preview-features"]
+= Managing preview features
```

Comment:
see later comment about "developer preview" and features, maybe if that is correct, change to "Managing technology preview features" ?